public/index.html

```diff
@@ -91,7 +99,9 @@
             <div class="centered-content">
                 <div @click="goBack" class="back-button"><img src="assets/chevron-left.svg">Tagasi</div>
                 <div id="result">
-                    <div v-if="currentClimber" :class="(currentClimber.certificate==='red') ? 'header red':'header green'">{{ currentClimber.certificate==='red' ? 'PUNANE' : 'ROHELINE' }} KAART</div>
+                    <div v-if="currentClimber"
+                        :class="(currentClimber.certificate==='red') ? 'header red':'header green'">{{
+                        currentClimber.certificate==='red' ? 'PUNANE' : 'ROHELINE' }} KAART</div>
```

Comment:
It's Estonian on mobile, but English on desktop. Is that intended?

Reply:
Fixed in another PR